### DISCUSSION TITLE
concatenating conditions

### DISCUSSION THREAD

**Discussion by pr-apes** [speedata/publisher#378] [Q&A] [answered]
@pgundlach,

can two conditions based on page number be concatenated such as in the following sample?

``` xml
<Pagetype name="odd page" test="sd:current-page() > 1 and sd:even(sd:current-page())">
```

Many thanks for your help.

**Comment by pgundlach** [comment, marked as answer]
Yes.
See https://doc.speedata.de/publisher/en/commandreference/pagetype/

There is an example, although I should re-format the page because the code is too wide.

**Reply by pr-apes** [reply]
I saw the example in full width, but I wondered whether this was limited to using different variables (not invoking `sd:current-page()` twice).

**Reply by pgundlach** [reply]
Sure you can concatenate anything you want. If the expression is covered by the (limited) xpath capabilities of the speedata Publisher, then you should be able to do so.
